<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="41" started-at="2020-06-25T06:35:26Z" finished-at="2020-06-25T06:35:26Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="41" started-at="2020-06-25T06:35:26Z" finished-at="2020-06-25T06:35:26Z">
      <class name="com.Sigma.TCCancelRun">
        <test-method status="PASS" signature="setup()[pri:0, instance:com.Sigma.TCCancelRun@76707e36]" name="setup" is-config="true" duration-ms="29587" started-at="2020-06-25T12:04:57Z" finished-at="2020-06-25T12:05:26Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method status="FAIL" signature="CancelRun(java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.Sigma.TCCancelRun@76707e36]" name="CancelRun" duration-ms="0" started-at="2020-06-25T12:05:26Z" data-provider="dp" finished-at="2020-06-25T12:05:26Z">
          <exception class="java.lang.RuntimeException">
            <message>
              <![CDATA[java.lang.NullPointerException]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.RuntimeException: java.lang.NullPointerException
	at org.testng.internal.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:161)
	at org.testng.internal.Parameters.handleParameters(Parameters.java:429)
	at org.testng.internal.Invoker.handleParameters(Invoker.java:1383)
	at org.testng.internal.Invoker.createParameters(Invoker.java:1075)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1180)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
Caused by: java.lang.NullPointerException
	at com.src.test.java.com.utilities.ExcelReading.getRowCount(ExcelReading.java:58)
	at com.Sigma.Testbase.getdata(Testbase.java:147)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:135)
	... 20 more
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.RuntimeException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- CancelRun -->
        <test-method status="PASS" signature="close()[pri:0, instance:com.Sigma.TCCancelRun@76707e36]" name="close" is-config="true" duration-ms="1" started-at="2020-06-25T12:05:26Z" finished-at="2020-06-25T12:05:26Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- close -->
      </class> <!-- com.Sigma.TCCancelRun -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
